u2-rounting-controllers/features/rgbController.js

```diff
@@ -0,0 +1,20 @@
+const getRGB = (req, res) => {
+  const r = parseInt(req.query.r) || 0
+  const g = parseInt(req.query.g) || 0
+  const b = parseInt(req.query.b) || 0
+  const average = Math.round((r + g + b) / 3)
+
+  if (average < 128) {
+    res.send(`<html><body style="background-color: rgb(${r}, ${g}, ${b});">
+    <font color=white> red: ${r}, green: ${g}, blue: ${b} 
+    avg: ${average} </font></body></html>`)
+  } else {
+    res.send(`<html><body style="background-color: rgb(${r}, ${g}, ${b});">
+    <font color=black> red: ${r}, green: ${g}, blue: ${b} 
```

Comment:
ฉันชอบที่คุณระบุสีที่นี่ด้วย! การตั้งค่าทั้งสีพื้นหลังและพื้นหน้าร่วมกันนั้นดีสำหรับการเข้าถึง (เช่น ความคมชัดของสีที่ดี)
I like that you specified the color here too! Setting both background & foreground colors together helps to ensure accessibility (e.g. good color contrast).